tests/resources/test_file_link.py

```diff
@@ -407,6 +407,11 @@ def test_process_file(collection, session):
                                 'type': 'integer_bounds',
                                 'lower_bound': 0,
                                 'upper_bound': 10
+                            },
+                            'exact_range_bounds': {
+                                'type': 'integer_bounds',
+                                'lower_bound': 0,
+                                'upper_bound': 10
```

Comment:
If I'm following the intent here, shouldn't we have the exact and recommended values be different to make sure things are plumbed correctly?

Reply:
Well we'e relying directly on the serialization stuff, I'm not concerned about a mismatch since we don't otherwise interact with it. We're well tested in the backend (and e2e's), so I think this is fine.

Reply:
I'll trust your instinct.